marathon/README.md

```diff
@@ -31,6 +31,60 @@ The Marathon check is included in the [Datadog Agent][1] package, so you don't n
 
 2. [Restart the Agent][4] to begin sending Marathon metrics to Datadog.
 
+#### Log Collection
+
+ **Available for Agent >6.0**
+
+ 1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:
+
+     ```yaml
+    logs_enabled: true
+    ```
+
+ 2. Because Marathon uses logback, you can specify a custom log format. With Datadog, two formats are supported out of the box, the default one provided by Marathon, and the Datadog recommended format. Add a fileappender to your configuration as in the following example and replace `$PATTERN$` with either `[%date] %-5level %message \(%logger:%thread\)%n` for Marathon default format or `%d{yyyy-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n` for Datadog recommended format.
```

Comment:
Since logback is used, should we also suggest using logback `json` appender too ? Using https://github.com/logstash/logstash-logback-encoder for example.

That way, the user can add any custom attribute they want. Of course, we need some guideline on how some fields should be named so they are mapped correctly.

@NBParis Any though on this ? Is there somewhere a guideline on how to integrate well logback json output with Datadog logs backend ?

Reply:
I would actually keep it the way it is. 
Mostly because the `message` attribute has a lot of impacts (like being free text searchable or being used for patterns).

So in general, logging in JSON is mostly for when you know what you do and already have a logic in place. So let's keep the suggestion as is.